x-pack/plugins/security_solution/common/asset_criticality/constants.ts

```diff
@@ -5,9 +5,65 @@
  * 2.0.
  */
 
+import { i18n } from '@kbn/i18n';
+import { euiLightVars } from '@kbn/ui-theme';
+import type { AssetCriticalityRecord } from '../api/entity_analytics/asset_criticality';
+
 export const ASSET_CRITICALITY_INDEX_PATTERN = '.asset-criticality.asset-criticality-*';
 
 type AssetCriticalityIndexPrivilege = 'read' | 'write';
 export const ASSET_CRITICALITY_REQUIRED_ES_INDEX_PRIVILEGES = {
   [ASSET_CRITICALITY_INDEX_PATTERN]: ['read', 'write'] as AssetCriticalityIndexPrivilege[],
 };
+
+export const PICK_ASSET_CRITICALITY = i18n.translate(
+  'xpack.securitySolution.timeline.sidePanel.hostDetails.assetCriticality.pick',
+  {
+    defaultMessage: 'Pick asset criticality level',
+  }
+);
+
+export const CREATE_ASSET_CRITICALITY = i18n.translate(
+  'xpack.securitySolution.timeline.sidePanel.hostDetails.assetCriticality.create',
+  {
+    defaultMessage: 'No criticality assigned yet',
+  }
+);
+
+export const ASSET_CRITICALITY_OPTION_TEXT: Record<
+  AssetCriticalityRecord['criticality_level'],
+  string
+> = {
+  normal: i18n.translate(
+    'xpack.securitySolution.timeline.sidePanel.hostDetails.assetCriticality.pickerOption.normal',
+    {
+      defaultMessage: 'Entity risk score rises at normal speed',
+    }
+  ),
+  not_important: i18n.translate(
+    'xpack.securitySolution.timeline.sidePanel.hostDetails.assetCriticality.pickerOption.notImportant',
+    {
+      defaultMessage: 'Entity risk score rises slower',
+    }
+  ),
+  important: i18n.translate(
+    'xpack.securitySolution.timeline.sidePanel.hostDetails.assetCriticality.pickerOption.important',
+    {
+      defaultMessage: 'Entity risk score rises faster',
+    }
+  ),
+  very_important: i18n.translate(
+    'xpack.securitySolution.timeline.sidePanel.hostDetails.assetCriticality.pickerOption.veryImportant',
+    {
+      defaultMessage: 'Entity risk score rises much faster',
+    }
+  ),
+};
+
+export const CRITICALITY_LEVEL_COLOR: Record<AssetCriticalityRecord['criticality_level'], string> =
+  {
+    very_important: '#E7664C',
+    important: '#D6BF57',
+    normal: '#54B399',
+    not_important: euiLightVars.euiColorMediumShade,
+  };
```

Comment:
This is styling related, it can be defined in the same component file.

Reply:
hmmm, not so sure, I suspect this will probably apply to any place where we want to display asset criticality levels, no? 
There's a similar thing going on for `RiskSeverity`, which is where I actually got some of the colors

Reply:
Discussed with @machadoum, leaving it within the components file but in an asset criticality folder

Reply:
Okay, I was also thinking about an `asset_criticality` directory. Sounds good